### DISCUSSION TITLE
There is any way to add characters from another account under the same battle net?

### DISCUSSION THREAD

**Discussion by 20kilu** [SavedInstances/SavedInstances#765] [General]
I use 2 accounts under the same battle net, one for AH and storage and the main one, there is any way to make the addon share this information with this bank like some other addons do?

Thanks in advance
20kilo

**Comment by LiangYuxuan** [comment]
#364 , and if you find one addon that can do such things, please share.

**Comment by 20kilu** [comment]
TSM have an option called Account Syncing and its done though ingame whisper if I'm not mistaken, works wonders to keep up all my sales in the my account!
